app/services/mhv/user_account/creator.rb

```diff
@@ -30,16 +31,17 @@ def perform
       private
 
       def create_mhv_user_account!
+        return mhv_account_creation_response if mhv_account_creation_response.nil? && from_cache_only
```

Comment:
Is there a reason this doesn't return a `MHVUserAccount`?

Reply:
I changed it to explicitly return `nil`. if there is nothing in the cache and we only want to pull from the cache we return `nil`, otherwise we try to create an `MHVUserAccount`